conan/tools/_compilers.py

```diff
@@ -56,6 +56,25 @@ def architecture_flag(settings):
     return ""
 
 
+def build_type_link_flags(settings):
+    """
+    returns link flags specific to the build type (Debug, Release, etc.)
+    [-debug]
+    """
+    compiler = settings.get_safe("compiler")
+    build_type = settings.get_safe("build_type")
+    if not compiler or not build_type:
+        return []
+
+    # https://github.com/Kitware/CMake/blob/d7af8a34b67026feaee558433db3a835d6007e06/
+    # Modules/Platform/Windows-MSVC.cmake
+    if compiler in ["msvc", "Visual Studio"]:
+        if build_type in ("Debug", "RelWithDebInfo"):
+            return ["-debug"]
+
+    return []
```

Comment:
Perhaps, we could keep it as simple like this one for now:

```python
def build_type_link_flags(settings):
    """
    returns link flags specific to the build type (Debug, Release, etc.)
    [-debug]
    """
    compiler = settings.get_safe("compiler")
    build_type = settings.get_safe("build_type")
    # https://github.com/Kitware/CMake/blob/d7af8a34b67026feaee558433db3a835d6007e06/
    # Modules/Platform/Windows-MSVC.cmake
    if compiler in ["msvc", "Visual Studio"] and build_type in ("Debug", "RelWithDebInfo"):
        return ["-debug"]
    else:
        return []
```

Reply:
Yes, could be. Current structure is parallel to the other "flags" methods, I mostly copied ``build_type_flags()``, probably better to keep the same method structure? Even if this is a bit simpler now, it might get other compiler flags or something like that?